src/apache/conf/httpd.conf

```diff
@@ -193,3 +193,38 @@ ServerSignature Off
 <IfDefine EnableHTTPS>
     Include ${SNAP}/conf/ssl.conf
 </IfDefine>
+
+<IfDefine EnablePrettyurls>
+    # From here: https://github.com/nextcloud/server/blob/stable21/lib/private/Setup.php#L523-L561
```

Comment:
Shouldn't we be using the `stable20` version?

Reply:
I thought thid would make sense to implement with NC21? Or do you wanna backport it?

Reply:
Well, there's still no ETA for the snap using nextcloud 21, so I'm not sure what will come first: this functionality or that

Reply:
Okay, I mean as I've stated at the very top of this issue is that probably extracting the specific lines of code out of the nextcloud tarball would probably the best way of doing it . But I don't know if that is posibble with snaps...

Reply:
Yes, I think that it can be doable, I just don't know exactly how (maybe you can ask for help on https://forum.snapcraft.io/)...meanwhile, to test if this works, I think that we should adapt it to the current latest version nextcloud on the snap, 20 in this case

Reply:
I've used the config now from stable20 but it seems like nothing changed concerning the config between those two versions.

Reply:
I think maybe it makes sense to not run the script from the snapcract.yml but maybe via github actions. This would allow to see any change to the file on new PRs.

Reply:
I'll try to figure out how to do that.

Reply:
should be done